src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -22,39 +22,77 @@
 
     private static final String TAG_NATIVE = "SyncthingNativeCode";
 
+    public static final String UNIT_TEST_PATH = "was running";
+
     private final Context mContext;
 
-    private String mCommand;
+    private boolean mGenerate;
+
+    private Command mCommand;
+
+    private String mManualCommand;
 
+    public enum Command {
+        generate, // Generate keys, a config file and immediately exit.
+        main,     // Run the main Syncthing application.
+        reset,    // Clear the Syncthing index.
+        manual,   // Manual command
```

Comment:
Okay that's right, but have you ever used that? (I haven't) And as long as there's no button for these, we might as well leave it out.